app/factories/post_factory.rb

```diff
@@ -13,8 +13,9 @@ class << self
     # Params:
     # +post_contents+::markdown in a given post
     # +file_path+::the path on GitHub to the post
```

Comment:
Add the `ref` as a parameter to this method comment.

Reply:
Look below it's there